js/libs.js

```diff
@@ -380,7 +382,7 @@ Fliplet.Registry.set('comflipletanalytics-report:1.0:core', function(element, da
   }
 
   function attachEventListeners() {
-
+    // eslint-disable-next-line spaced-comment
```

Comment:
why? can't you just fix the source code?

Reply:
@squallstar 
Fixed the source code as lint recommended